.github/workflows/cppcmake.yml

```diff
@@ -35,8 +35,8 @@ jobs:
         - {name: "ubuntu-20.04", os: "ubuntu-20.04",   cmake_extra: "-DLSL_BUNDLED_PUGIXML=OFF"}
         - {name: "ubuntu-18.04", os: "ubuntu-latest",  docker: "ubuntu:18.04" }
         - {name: "ubuntu-22.04", os: "ubuntu-22.04", cmake_extra: "-DLSL_BUNDLED_PUGIXML=OFF" }
-        - {name: "windows-x64",  os: "windows-2019", cmake_extra: "-T v140,host=x86"}
-        - {name: "windows-32",   os: "windows-2019", cmake_extra: "-T v140,host=x86 -A Win32"}
+        - {name: "windows-x64",  os: "windows-2019", cmake_extra: "-T host=x86"}
```

Comment:
I generally agree with this, but the [compatibility page](https://labstreaminglayer.readthedocs.io/dev/build_env.html?highlight=visual#recommended-compiler-toolchain) still lists VS 2015 as supported.

As a conservative approach, we could update the docs and change it to v141.
Or we add a VS 2017 runner to the manually run configurations (like static mingw) and let the CI system use the latest toolchain available. I'm not up to date on the latest msvc compatibility issues, but this might mean that end users will need a newer msvcruntime dll and some older Windows versions might get dropped.